src/app/App.js

```diff
@@ -0,0 +1,43 @@
+import {useState} from 'react';
+import './App.css';
+import Counter from '../shared/ui/Counter/Counter';
+import SearchForm from '../shared/ui/SearchForm/SearchForm';
+import SelectTabs from '../shared/ui/SelectTabs/SelectTabs';
```

Comment:
to avoid name duplication in imports we can create an `index.js` inside `shared/ui` folder and export your components like:
`export { SelectTabs } from './SelectTabs';`

here is an example: 
https://github.com/yesnoruly/loripsum-generator/tree/master/src/shared/ui